### ISSUE TITLE
WPF example application not working anymore

### ISSUE THREAD

**Issue by tbrbde** [JKorf/Binance.Net#193]
For me the WPF example is working using older versions ("Binance.Net" version="3.1.5" and "CryptoExchange.Net" version="0.0.25"). With the latest versions (3.2.6 and 0.0.44) I get the error "API-key format invalid" when I try to enter API key and secret in the WPF application settings dialog. I tried it several times with the same key and secret as they are working with the old versions.

**Comment by JKorf** [comment]
Hi, it's been a while since I've updated the Examples, I'll update them to work with the latest versions again.

**Comment by JKorf** [comment]
I've updated the examples, but I don't know why you would get that error message with the earlier version. Can you try again to see if it works now though?

**Comment by tbrbde** [comment]
I downloaded and tested the updated version. I started the WPF application and clicked on the settings symbol in order to open the settings dialog. On the settings dialog I entered the api key and secret. Then I get the error message "Error requesting data: API-key format invalid". I confirm with OK. And now I get a second error message: "Error requesting data: No credentials provided for private endpoint." Then I tried it again with the older version and got no error message. Key and secret are working with the older version.
